sdk/servicebus/azure-servicebus/samples/async_samples/send_queue_async.py

```diff
@@ -21,20 +21,25 @@
 
 
 async def send_single_message(sender):
-    message = Message("DATA" * 64)
-    await sender.send_messages(message)
+    message = Message("Single Message")
+    await sender.send(message)
+
+
+async def send_a_list_of_messages(sender):
+    messages = [Message("Message in list") for _ in range(10)]
+    await sender.send(messages)
 
 
 async def send_batch_message(sender):
     batch_message = await sender.create_batch()
-    while True:
+    for _ in range(10):
         try:
-            batch_message.add(Message("DATA" * 256))
+            batch_message.add(Message("Message inside a BatchMessage"))
         except ValueError:
             # BatchMessage object reaches max_size.
             # New BatchMessage object can be created here to send more data.
             break
-    await sender.send_messages(batch_message)
+    await sender.send(batch_message)
```

Comment:
```suggestion
    await sender.send_messages(batch_message)
```

Reply:
my fault, thanks for double-checking -- I trust the git but now it doesn't worth my trust again ><